tests/dragonfly/cluster_test.py

```diff
@@ -305,8 +316,11 @@ async def test_cluster_slot_ownership_changes(df_local_factory: DflyInstanceFact
 
 
 # Tests that master commands to the replica are applied regardless of slot ownership
+@pytest.mark.parametrize("set_cluster_node_id", [True, False])
 @dfly_args({"proactor_threads": 4, "cluster_mode": "yes"})
-async def test_cluster_replica_sets_non_owned_keys(df_local_factory):
+async def test_cluster_replica_sets_non_owned_keys(
+    df_local_factory: DflyInstanceFactory, set_cluster_node_id: bool
```

Comment:
you dont use this param in the test

Reply:
Sorry, leftovers. I decided to use the test below instead (`test_cluster_native_client`)